src/lib/components/shared/markdown.svelte

```diff
@@ -52,16 +44,22 @@
         mangle(),
         // Marked defaults
         {
-            renderer,
+            renderer: {
+                del(text: string) {
+                    return `<del class='unstyled'>${text}</del>`;
+                }
+            },
             // We dont need github like header prefix
             headerIds: false
         }
     );
 
-    let html: string;
-    $: html = sanitize(marked.parse(markdown));
+    let promise_html: Promise<string>;
+    $: promise_html = sanitize(marked.parse(markdown));
 </script>
 
-<markdown class={klass}>
-    {@html html}
-</markdown>
+{#await promise_html then html}
+    <markdown class={klass}>
+        {@html html}
+    </markdown>
+{/await}
```

Comment:
We can add skeleton like loader here later? lol no need I think... well LGTM

Reply:
Not necessary in my opinion